### ISSUE TITLE
NTR: MR Spectroscopy: succinate high in brain (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8890]
New term request MR Spectroscopy: succinate high in brain (segal_081106161453): MRS

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000917
- label: Elevated brain succinate level by MRS
- parents: [Abnormal metabolic brain imaging by MRS (HP:0012705)](https://hpo.jax.org/app/browse/term/HP:0012705)
- pmid: 
- def: An increase in the level of succinate in the brain identified by magnetic resonance spectroscopy (MRS).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8890
- synonyms: Elevated brain succinate level by magnetic resonance spectroscopy(NONE)
